Execute script (#3837)
edgedb/edgedb: docs/reference/protocol/index.rst

    expected to adjust the request data and send
    :ref:`ref_protocol_msg_execute` again.

Each of the messages could contain one or more EdgeQL commands in the same text
string. If more than one EdgeQL command is found in a single message, the
server will treat the commands as an EdgeQL script. EdgeQL scripts are always
atomic, they will be executed in an implicit transaction block if no explicit
transactions are started beforehand. Therefore, EdgeQL scripts have limitations
on the EdgeQL commands they could contain:

* Transaction control commands are not allowed, like ``start transaction``,

>   ``commit``, ``rollback to savepoint`` or ``commit migration``.

Reviewer: ```suggestion
  ``commit``, ``declare savepoint``, or ``rollback to savepoint``.
```

> * Non-transactional commands are not allowed, like ``create database``.
> * ``configure instance`` command is not allowed.

Reviewer: ```suggestion
* Non-transactional commands, like ``create database`` or ``configure instance`` are not allowed.
```
